src/h5web/visualizations/shared/AxisSystem.tsx

```diff
@@ -26,12 +26,12 @@ interface Props {
 function AxisSystem(props: Props): JSX.Element {
   const { axisOffsets } = props;
 
+  const { abscissa, ordinate } = useContext(AxisSystemContext);
   const { camera, size } = useThree();
   const { width, height } = size;
 
-  const { abscissaConfig, ordinateConfig } = useContext(AxisSystemContext);
-  const abscissa = useAbscissaScale();
-  const ordinate = useOrdinateScale();
+  const abscissaScale = getAxisScale(abscissa, width);
+  const ordinateScale = getAxisScale(ordinate, height);
```

Comment:
Note that this hook-less implementation is now consistent with the way we call `getTicksProp` below: we pass the axis info object as first parameter.